client/my-sites/importer/importer-logo.jsx

```diff
@@ -19,34 +19,37 @@ import SocialLogo from 'components/social-logo';
  */
 import './importer-logo.scss';
 
-const ImporterLogo = ( { icon } ) => {
+const ImporterLogo = ( { icon, className = 'importer__service-icon' } ) => {
+	/* eslint-disable wpcalypso/jsx-classname-namespace */
+
+	if ( 'blogger' === icon ) {
+		icon = 'blogger-alt';
+	}
+
 	if ( includes( [ 'wordpress', 'blogger-alt', 'squarespace' ], icon ) ) {
-		return <SocialLogo className="importer__service-icon" icon={ icon } size={ 48 } />;
+		return <SocialLogo className={ className } icon={ icon } size={ 48 } />;
 	}
 
 	if ( 'wix' === icon ) {
-		return <WixLogo />;
+		return <WixLogo className={ className } />;
```

Comment:
Why this change? (and similar ones for other services)

Reply:
I was hoping to keep the markup tidy by giving these components classes that were consistent with their parents in the import preview. I also thought it would help avoid having to override existing CSS. But the change made the components less simple, and it seems the linter has no problem with me using the existing classnames, so I've removed these params, thanks!